migrations/0017_create_tickets.up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS tickets (
```

Comment:
I think "tickets" -> "rsvp" is a little confusing. but I also don't love "reservation_types" like I was using. maybe "openings"? "spots"?

Reply:
yeahhh couldn't come up with anything satisfying either

Reply:
`spots` seems good to me